api/src/test/java/org/pmiops/workbench/api/ProfileControllerTest.java

```diff
@@ -120,14 +120,15 @@
   private static final String CURRENT_POSITION = "Tester";
   private static final String FAMILY_NAME = "Bobberson";
   private static final String GIVEN_NAME = "Bob";
+  private static final String GSUITE_DOMAIN = "Bob";
```

Comment:
To make the errors or debug info clearer, I'd make this look more like a domain, e.g. "bobby.com".

Reply:
Yep, that was copy-pasta mistake, done.